CRM/Utils/Date.php

```diff
@@ -458,8 +461,10 @@ public static function customFormat($dateString, $format = NULL, $dateParts = NU
         '%p' => strtolower($type),
         '%P' => $type,
         '%Y' => $year,
+        '%y' => substr($year, 2),
         '%s' => str_pad($second, 2, 0, STR_PAD_LEFT),
         '%S' => str_pad($second, 2, 0, STR_PAD_LEFT),
+        '%Z' => date('T'),
```

Comment:
This needs to take the date being custom-format'd into account, since otherwise it always gives the timezone corresponding to "now".

Reply:
ok updated to that now @demeritcowboy good pick up